### ISSUE TITLE
BMI not updating?

### ISSUE THREAD

**Issue by jcw** [killercup/cycle-webpack-starter#2]
Total FRP & cycle rookie here: sliders are working, but BMI is not updating? I can see that calculateBMI is not being called when the sliders are changed. Will investigate (and learn a lot from it), but any tips on how to go about this would be much appreciated.

**Comment by killercup** [comment]
I copied the example from cycle's documentation and then changed it to use
JSX. (That was before the docs used JSX as well.)

Jean-Claude Wippler notifications@github.com schrieb am Fr., 24. Juli
2015 um 18:00:

**Comment by jcw** [comment]
It works correctly when compiled (with index.html copied to build/ and the webpack script removed).
But not in dev mode with `npm start` and Chrome or Safari as browser.

**Comment by killercup** [comment]
Thanks for the bug report, sounds like an issue with webpack. I'll look
into it later today.

Jean-Claude Wippler notifications@github.com schrieb am Fr., 24. Juli
2015 um 18:27:

**Comment by killercup** [comment]
Fixed.

> sounds like an issue with webpack

Nope, webpack continues to be awesome. I've added types to the `calculateBMI` function to test how Babel handles TypeScript/Flow style annotations with the [typecheck plugin](https://github.com/codemix/babel-plugin-typecheck). And it catches an error! The labeled-slider events contain strings for values, not integers. Sadly, the thrown errors are not shows anywhere and just get swallowed by RxJS. I'll have to see how these can be handled better.
